test failure on centos7

## Version of Singularity:

master

```
--- FAIL: TestConfigure (0.00s)
    --- FAIL: TestConfigure/with_unconfined_SELinux_context (0.00s)
        security_test.go:126: unexpected failure with unconfined SELinux context: write /proc/self/task/20449/attr
/exec: invalid argument
```

@cclerget 

Ideas on why this is failing on centos7?

--- a/internal/pkg/security/security_test.go
+++ b/internal/pkg/security/security_test.go
@@ -84,7 +84,7 @@ func TestConfigure(t *testing.T) {
 			desc: "with unconfined SELinux context",
 			spec: specs.Spec{
 				Process: &specs.Process{
-					SelinuxLabel: "unconfined_u:unconfined_r:unconfined_t",
+					SelinuxLabel: "unconfined_u:unconfined_r:unconfined_t:s0",
 				},
 			},
 			disabled: !selinux.Enabled(),

